specification/cognitiveservices/data-plane/Face/stable/v1.0/Face.json

```diff
@@ -3805,10 +3805,19 @@
     },
     "RecognitionModel": {
       "type": "string",
-      "description": "Recognition model name, maximum length is 128.",
-      "maxLength": 128
+      "description": "Recognition model name.\nRecognition model is used when the face features are extracted, so a recognition model version could be provided when performing a Detection.\n",
+      "default": "Recognition_V01",
+      "x-nullable": false,
+      "x-ms-enum": {
+        "name": "RecognitionModel",
+        "modelAsString": false
+      },
+      "enum": [
+        "Recognition_V01",
```

Comment:
We expect user to pass `recognition_v01` to us, even though we will ignore the cases. So please double check do we need to change these enum values as lower case.

Reply:
Done.